.github/workflows/validate.yml

```diff
@@ -8,54 +13,38 @@ jobs:
       # checkout code
       - name: Checkout
         uses: actions/checkout@v3
+        with:
+          path: project
+          fetch-depth: 2
       # checkout schema as xmllint can't https 
       - name: Checkout schema repo
         uses: actions/checkout@v3
         with:
           repository: BetaMasaheft/Schema
           path: schema
-          fetch-depth: 1  
       - name: Install Test Dependencies
+        run: sudo apt-get install libxml2-utils
+      - name: Check for changed source files
+        id: changed
         run: |
-          sudo apt-get update 
-          sudo apt-get install libxml2-utils
-          xmllint --version
-      # - name: check local Files
-      #   run: |
-      #     cd schema
-      #     ls
-      # - name: Build Xar Package
-      #   run:  ant
-      - name: Validate XML Files
+          echo "::set-output name=files::$( \
+            git diff --name-only --diff-filter=ACMRT \
+            ${{ github.event.pull_request.base.sha }} \
+            ${{ github.sha }} -- './**/*.xml' | xargs)"
```

Comment:
-- '!./**/transkribus.xml,!./**/facsimile.xml,./**/*.xml' could do the trick

Reply:
earlier you used `transkribus*.xml` I assume we want to similarly exclude facs1.xml, facs.xml, …